snippets/LimitedTextarea.md

```diff
@@ -46,7 +51,7 @@ ReactDOM.render(
   document.getElementById('root')
 );
 ```
-
+Sandbox Link: https://codesandbox.io/s/5z2j52zvon
```

Comment:
No links to external websites should be added, hence you should delete this, too. We will eventually set up a website where you can see all snippets in action.